tools/test/windows/tw.cc

```diff
@@ -232,12 +232,43 @@ void LogErrorWithValue(const int line, const std::wstring& msg, DWORD value) {
   }
 }
 
+void LogErrorWithArg(const int line, const std::string& msg,
+                     const std::string& arg) {
+  std::stringstream ss;
+  ss << msg << " (arg: " << arg << ")";
+  LogError(line, ss.str());
+}
+
+void LogErrorWithArg(const int line, const std::string& msg,
+                     const std::wstring& arg) {
+  std::string acp_arg;
+  if (blaze_util::WcsToAcp(arg, &acp_arg)) {
+    LogErrorWithArg(line, msg, acp_arg);
+  }
+}
+
+void LogErrorWithArg2(const int line, const std::string& msg,
```

Comment:
Nit: How about `LogErrorWithArgW`?

Reply:
It's now about wstring vs. string, just that there's an extra argument. So I prefer this name.